### PR TITLE
opam 2.0.8 release blog post

### PR DESCRIPTION
**Pull request by rjbou**

### CODE REVIEW

File: opam-2-0-8.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * Add colon for fish MANPATH fix. [[#4084](https://github.com/ocaml/opam/pull/4084) [@rjbou](https://github.com/rjbou) - fix [#4078](https://github.com/ocaml/opam/issues/4078)]
+ * No error when linked directory doesn't exist (e.g. XDG defined) [[#4278](https://github.com/ocaml/opam/pull/4278) [@kit-ty-kate](https://github.com/kit-ty-kate)]
+ * Add quotes to avoid space unwanted behaviors [[#4278](https://github.com/ocaml/opam/pull/4278) [@kit-ty-kate](https://github.com/kit-ty-kate)]
+ * Handle `CCACHE_DIR` environment variable in sandbox script.  [[#4087](https://github.com/ocaml/opam/pull/4087) [@rjbou](https://github.com/rjbou) - fix [#4079](https://github.com/ocaml/opam/issues/4079)]
+ * Follow links of `~/.cache` & `~/.cache/dune` for bwrap call.  [[#4087](https://github.com/ocaml/opam/pull/4087) [@rjbou](https://github.com/rjbou) - fix [#4068](https://github.com/ocaml/opam/issues/4068)]
+ * Don't overwrite user's sandbow script modification. [#4020 [#4092](https://github.com/ocaml/opam/pull/4092) [@rjbou](https://github.com/rjbou)]
+ * On MacOS sandbox script, always read write mount `/tmp` [#3742 @rjbou - fix ocaml/opam-repository#13339]
```
Comment (avsm) [inline opam-2-0-8.md:18]:
```suggestion
 * On the macOS sandbox script, always read/write mount `/tmp` [#3742 @rjbou - fix ocaml/opam-repository#13339]
```

File: opam-2-0-8.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This new version contains some [backported](https://github.com/ocaml/opam/pull/4425) fixes:
+ * Add colon for fish MANPATH fix. [[#4084](https://github.com/ocaml/opam/pull/4084) [@rjbou](https://github.com/rjbou) - fix [#4078](https://github.com/ocaml/opam/issues/4078)]
+ * No error when linked directory doesn't exist (e.g. XDG defined) [[#4278](https://github.com/ocaml/opam/pull/4278) [@kit-ty-kate](https://github.com/kit-ty-kate)]
+ * Add quotes to avoid space unwanted behaviors [[#4278](https://github.com/ocaml/opam/pull/4278) [@kit-ty-kate](https://github.com/kit-ty-kate)]
+ * Handle `CCACHE_DIR` environment variable in sandbox script.  [[#4087](https://github.com/ocaml/opam/pull/4087) [@rjbou](https://github.com/rjbou) - fix [#4079](https://github.com/ocaml/opam/issues/4079)]
+ * Follow links of `~/.cache` & `~/.cache/dune` for bwrap call.  [[#4087](https://github.com/ocaml/opam/pull/4087) [@rjbou](https://github.com/rjbou) - fix [#4068](https://github.com/ocaml/opam/issues/4068)]
+ * Don't overwrite user's sandbow script modification. [#4020 [#4092](https://github.com/ocaml/opam/pull/4092) [@rjbou](https://github.com/rjbou)]
```
Comment (avsm) [inline opam-2-0-8.md:17]:
```suggestion
 * Don't overwrite user's sandbox script modifications. [#4020 [#4092](https://github.com/ocaml/opam/pull/4092) [@rjbou](https://github.com/rjbou)]
```

File: opam-2-0-8.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    opam update; opam install opam-devel
+    ```
+
+   (then copy the opam binary to your PATH as explained, and don't forget to run `opam init --reinit -ni` to enable sandboxing if you had version 2.0.0~rc manually installed or to update you sandbox script)
+
+3. From source, manually: see the instructions in the [README](https://github.com/ocaml/opam/tree/2.0.8#compiling-this-repo).
+
+We hope you enjoy this new minor version, and remain open to [bug reports](https://github.com/ocaml/opam/issues) and [suggestions](https://github.com/ocaml/opam/issues).
+
+> NOTE: this article is cross-posted on [opam.ocaml.org](https://opam.ocaml.org/blog/) and [ocamlpro.com](http://www.ocamlpro.com/category/blog/). Please head to the latter for the comments!
```
Comment (avsm) [inline opam-2-0-8.md:53]: perhaps link to discuss.ocaml.org as well for comments? Or we can just start doing that for the 2.1.0 series

**Comment by avsm** [comment]
lgtm just minor suggestions

File: opam-2-0-8.md (hunk 3 above)
Comment (rjbou) [reply opam-2-0-8.md:53]: I took a template, but yes, as we always post on discuss too, so better rely on that first

**Review by dra27 — APPROVED** [review]
LGTM too!

**Pull request merged**